cmd/zc_enumerator.go

```diff
@@ -486,18 +485,39 @@ func InitResourceTraverser(resource common.ResourceString, location common.Locat
 
 		recommendHttpsIfNecessary(*resourceURL)
 
-		bfsURL := azbfs.NewBfsURLParts(*resourceURL)
+		// Convert BlobFS pipeline to blob-compatible pipeline
+		var credElement azblob.Credential
+		if credential.CredentialType == common.ECredentialType.SharedKey() {
```

Comment:
I thought AzCopy doesnt support shared key?

Reply:
It does, actually! Only for DFS though.